fix(build): fix grafana build for go 1.16 (#1)
CleverCloud/deploy-grafana: README.md

# How it works

> 1. Use this repository as a JS clever cloud application. It will build and run grafana using the root `package.json` file. 

Reviewer: ```suggestion
1. Use this repository as a Node.js application. It will build and run Grafana using the root `package.json` file. 
```

> 2. Check the node version use in `package.json` (to set the grafana expected one, for Grafana 7.3.5 it's node version 12: https://github.com/grafana/grafana/blob/v7.3.5/package.json#L314 ).

Reviewer: ```suggestion
2. Check Node.js version used in `package.json`, for instance Grafana 7.3.5 expects Node.js 12 (see Grafana's [package.json](https://github.com/grafana/grafana/blob/v7.3.5/package.json#L314)).
```

3. This will build both the back-end and the front-end of Grafana.

> 4. Add ```GOPATH=/home/bas/<app_id>/go_home``` in your environment variables where <app_id> is the id of the app on clever-cloud.

Reviewer: ```suggestion
4. Add ```GOPATH=/home/bas/<app_id>/go_home``` in your environment variables where `<app_id>` is the id of the app on clever-cloud.
```

> 5. Finally add ```GF_SERVER_HTTP_PORT=8080``` in your app environment variables to ensure Grafana starts on the correct port and **make sure you have enabled a dedicated build instance**.

Reviewer: Can we explain why we have to use a dedicated build ?

Author: It's not, this PR goal is to repair the build version (for go1.16). I will do later a PR using Grafana official binaries

Reviewer: Fine :wink:
